Gemfile

```diff
@@ -33,6 +33,8 @@ gem 'jbuilder', '~> 2.5'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
+gem 'faraday'
+gem 'figaro'
```

Comment:
I don't think figaro is needed for this particular application since the API doesn't have a secret key.